books/RayTracingTheRestOfYourLife.html

```diff
@@ -47,22 +47,51 @@
 
 
 
-
 A Simple Monte Carlo Program
 ====================================================================================================
+<div class='together'>
+Let’s start with one of the simplest Monte Carlo programs. If you're not familiar with Monte Carlo
+programs, then it'll be good to pause and catch you up. There are two kinds of randomized
+algorithms: Monte Carlo algorithms and Las Vegas algorithms. Randomized algorithms can be found
+everywhere in computer graphics, so getting a decent foundation isn't a bad idea. A randomized
+algorithm is an algorithm that uses some amount of randomness in it's computation. A Las Vegas (LV)
+random algorithm always produces the correct result, whereas a Monte Carlo (MC) algorithm _may_
+produce a correct result (and frequently gets it wrong!). But for especially complicated problems
```

Comment:
Possibly “... _may_ produce a correct result, but more frequently is just close enough to produce an acceptable result.”

Reply:
I'm going to say no on this and the next comment.

The point of this sentence is to have an extremely simple boolean that differentiates LV from MC.
Namely: exactness.

the parenthesis at the end provides a slight lead-in to the comment at the end of the section, but I don't want to over complicate the point of this sentence.

The boolean.